### PR TITLE
feat: create datedive db and setup db with sqlalchemy

### PR DESCRIPTION
**Pull request by tmznwnel00**
image

### CODE REVIEW

File: server/model.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    __tablename__ = "User"
+    
+    id = Column(Integer, primary_key=True)
+    age = Column(Integer)
+    gender = Column(Enum('male', 'female'))
+    nickname = Column(String(15))
+    verified = Column(Boolean)
+    signup_date = Column(Date)
+    
+class User_photo(Base):
```
Comment (sungminoh) [inline server/model.py]: Class names should normally use the CapWords convention in python
https://peps.python.org/pep-0008/#class-names

**Review by sungminoh — APPROVED** [review]

File: server/model.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, Enum, Date, JSON
+from sqlalchemy.orm import relationship
+
+user = 'mysql'
+password = 'Abc1234!'
```
Comment (sungminoh) [inline server/model.py:8]: Later this should not be hard-coded
Instead, use env variable.

File: server/model.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+class User(Base):
+    __tablename__ = "User"
+    
+    id = Column(Integer, primary_key=True)
+    age = Column(Integer)
+    gender = Column(Enum('male', 'female'))
+    nickname = Column(String(15))
+    verified = Column(Boolean)
+    signup_date = Column(Date)
```
Comment (sungminoh) [inline server/model.py]: let's use `created_at` as same as other tables

File: server/model.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    
+class User_photo(Base):
+    __tablename__ = "User_photo"
+    
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer)
+    s3_path = Column(String)
+    photo_index = Column(Integer)
+    createdAt = Column(Date)
```
Comment (sungminoh) [inline server/model.py]: let's use `created_at` and `updated_at` for consistency.

File: server/model.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    updatedAt = Column(Date)
+    
+class Match(Base):
+    __tablename__ = "Match_table"
+    
+    id = Column(Integer, primary_key=True)
+    male_id = Column(Integer)
+    female_id = Column(Integer) 
+    status = Column(Enum('man waiting', 'man refuse', 'woman waiting', 'woman refuse', 'matched'))
```
Comment (sungminoh) [inline server/model.py:45]: On second thought, we may want to allow women to choose man cards later.
Thus, what about making it more generic e.g. `suggest`, `pass`, `waiting`, `reject`, `match`?

File: server/model.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    createdAt = Column(Date)
+    updatedAt = Column(Date)
+    
+class Questionnaire(Base):
+    __tablename__ = "Questionnaire"
+    
+    id = Column(Integer, primary_key=True)
+    description = Column(String)
+    options = Column(JSON)
```
Comment (sungminoh) [inline server/model.py:54]: array or json?

File: server/model.py (hunk 2 above)
Comment (tmznwnel00) [reply server/model.py:8]: oh I should modify user 'mysql' to 'test'. I will fix this.
Later, I will use dotenv

File: server/model.py (hunk 6 above)
Comment (tmznwnel00) [reply server/model.py:54]: I searched for mysql array type, **Text** and **JSON** can express **Array** type.

**Pull request merged**